conf/modules.config

```diff
@@ -289,6 +289,17 @@ process {
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
+        ext.args = [
+            "--score ${params.bgc_deepbgc_score}",
+            "--prodigal-meta-mode ${params.bgc_deepbgc_prodigalmetamode}",
```

Comment:
Good catch! But then it should be on `TRUE`  by default, because we already have `prodigal/prokka' default `-p meta`.

Reply:
Should also reuse the existing prodigal meta parameter we have for our 'own' prodigal process to activate this

Reply:
But @jfy133 `deepbgc` does not use the output from prodigal but rather does its own annotation, so we still need this parametre here.

Reply:
Which actually is something we should think about in the future for standardization purposes. It seems that multiple `modules' can take `.faa` files as input. So it will be better that we change all those tools to accept `faa` from either `prokka/bakta/prodigal` as input rather than the current `fna`, because each module will use diff annotations depending on the algorithm they use.

Reply:
> But @jfy133 deepbgc does not use the output from prodigal but rather does its own annotation on the input .fna, so we still need this parametre here.

But that's what I mean, we should make sure any execution of prodigial (either standalone in the pipeline or within a tool) use the same parameters, the data type would not change so we can assume the same parameters should be used.

> Which actually is something we should think about in the future for standardization purposes. It seems that multiple modules'  can take  '.faa'  files as input. So it will be better that we change all those tools to accept  'faa'  from either  prokka/bakta/prodigal` as input rather than the current 'fna', because each module will use diff annotations depending on the algorithm they use.

Oooh good point.. please make an issue!